cdc/owner/feed_state_manager.go

```diff
@@ -145,9 +175,13 @@ func (m *feedStateManager) handleAdminJob() (jobsPending bool) {
 			return
 		}
 		m.shouldBeRunning = true
+		m.lastErrorTime = time.Unix(0, 0)
```

Comment:
Please add some comments. Does it always clean up backoff?

Reply:
Before a fresh run, the backoff and lastErrorTime should be cleaned.